installer/web/app2.py

```diff
@@ -0,0 +1,56 @@
+import socket
+import webbrowser
+
+from flask import Flask, render_template, request
+
+from ..base import (
+    Account,
+    RaidenConfigurationFile,
+    Network,
+    build_infura_url,
+    RaidenClient,
+)
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["GET", "POST"])
+def install_raiden():
+    if request.method == "POST":
+        passphrase = request.form["keystore-pwd"]
+        project_id = request.form["proj-id"]
+
+        # Creates a keystore in my case on mac it is located in /users/taleldayekh/Library/Ethereum/keystore
+        account = Account.create(passphrase)
+
+        # Create a network object so we can grab the relevant network name. The network "goerli in this case" should be passed via radio button
+        network = Network("goerli")
```

Comment:
Use `Network.get_by_name` to ensure that you get an instance of the right Network class.